### DISCUSSION TITLE
Questions about the commands of nnUNetV2Runner

### DISCUSSION THREAD

**Discussion by Minxiangliu** [Project-MONAI/tutorials#1895] [Q&A] [answered]
Hi @KumoLiu ,

I'm new to using nnUNetV2Runner, I have some PET 3D images and I want to ask about a few things:

1. nnUNet does not seem to have images for simply training PET. Is this kind of data suitable for training?

2. In the introduction [here](https://github.com/Project-MONAI/tutorials/blob/main/nnunet/README.md#how-does-the-integration-work), I saw that 20 models will be trained by default. The command execution method is to use directly:
`python -m monai.apps.nnunet nnUNetV2Runner run --input_config='./input.yaml'`
or:
`python -m monai.apps.nnunet nnUNetV2Runner train --input_config "./input.yaml"`

**Comment by KumoLiu** [comment, marked as answer]
Hi @Minxiangliu, thanks for your interest here.
- `nnUNetV2Runner` is only a wrapper for nnUNet, so for more details about what can nnunet do for you, please refer to the nnunet docs.
https://github.com/MIC-DKFZ/nnUNet?tab=readme-ov-file#what-can-nnu-net-do-for-you
- For the command, you can refer to https://github.com/Project-MONAI/tutorials/blob/main/nnunet/README.md#run-nnu-net-modules-using-nnunetv2runner. Should be `train` instead of `run`.

Hope it helps.
